schemas/lists/units.json

```diff
@@ -99,6 +98,7 @@
       "mass-pound",
       "mass-stone",
       "mass-ton",
+      "mile",
```

Comment:
Why adding this?

Reply:
`mile` was missing but the earlier version of AJV didn't fail as `"$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",` use to take precedence on enums.